binr/rahash2/rahash2.c

```diff
@@ -288,10 +288,80 @@ int is_power_of_two(const ut64 x) {
 	return x && !(x & (x - 1));
 }
 
+int encrypt_or_decrypt (const char *algo, bool to_encrypt, const char *hashstr, int hashstr_len) {
+	bool no_key_mode = !strcmp ("base64", algo) || !strcmp ("base91", algo); //TODO: generalise this for all non key encoding/decoding.
```

Comment:
ISSUE 1: Should I add a function in a specific file this or any other like crypto.c which just return the boolean value if given algorithm needs a key or not? I am not sure where (file) to add it.